packages/nimble-components/build/generate-workers/source/render-worker.ts

```diff
@@ -0,0 +1,25 @@
+import { expose } from 'comlink';
+
+/**
+ * RenderWorker class is meant to be used within a Web Worker context, 
+ * using Comlink to facilitate communication between the main thread and the worker. 
+ * The RenderWorker class manages a matrix of dies, once an instance of RenderWorker is created, 
+ * it is exposed to the main thread using Comlink's `expose` method.
+ * This setup is used in the wafer-map component to perform heavy computational duties
+ */
+export class RenderWorker {
+    public worker!: number;
+    public dieMatrix: Uint8Array = Uint8Array.from([]);
+
+    public emptyMatrix(): void {
+        this.dieMatrix = Uint8Array.from([]);;
+    }
+
+    public updateMatrix(
+        data: Iterable<number>
+    ): void {
+        this.dieMatrix = Uint8Array.from(data);
+    }
+}
+const worker = new RenderWorker();
```

Comment:
Instead of exposing an instance of the class, lets expose the class itself.
See: https://github.com/GoogleChromeLabs/comlink/blob/dffe9050f63b1b39f30213adeb1dd4b9ed7d2594/tests/same_window.comlink.test.js#L195

Reply:
did it 
![image](https://github.com/ni/nimble/assets/110180309/1f10e45a-421f-4854-b189-b1a56cfa1abd)